src/uu/chmod/src/chmod.rs

```diff
@@ -195,21 +195,24 @@ impl Chmoder {
                         filename.quote()
                     );
                     if !self.quiet {
-                        return Err(USimpleError::new(
+                        show!(USimpleError::new(
                             1,
                             format!("cannot operate on dangling symlink {}", filename.quote()),
                         ));
                     }
+                    set_exit_code(1);
+                    continue;
```

Comment:
This is quite subjective, but in my opinion it becomes clearer if we remove these 2 lines. It simplifies the control flow a bit because there's only one `continue` in that case.

Reply:
Yeah you are for sure 100% right, I  guess i duplicated it while looking at some many `if` and `else if`. But thanks for noticing it!